clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -7337,8 +7337,8 @@ static void handleHLSLResourceBindingAttr(Sema &S, Decl *D,
   // Validate.
   if (!Slot.empty()) {
     switch (Slot[0]) {
-    case 'u':
     case 'b':
+    case 'u':
```

Comment:
Why make this change? if its for alphabetical reasons, stu should be the order, if not why leave as is?